Makefile

```diff
@@ -2,28 +2,53 @@
 ## Install gcn2scimma
 ##
 INSTALL_DIR   := /usr/local/gcn2hop
-PYTHON_FILES  := gcn2hop Utils.py 
+WORKING_DIR	  := ./gcn2hop
+PYTHON_FILES  := gcn2hop,Utils.py
 INSTALL_FILES := $(patsubst %,$(INSTALL_DIR)/%,$(PYTHON_FILES))
-
-.PHONY: all install install_dir test
+PYTHON = python
+
+.PHONY: all install test
+
+.PHONY: help
+help :
+	@echo
+	@echo 'Commands:'
+	@echo
+	@echo '  make test                  run unit tests'
+	@echo '  make lint                  run linter'
+	@echo '  make format                run code formatter, giving a diff for recommended changes'
+	@echo '  make doc                   make documentation'
+	@echo '  make install				install module
+	@echo
 
 all:
 	@echo No default make command. To install run "make install"
 
-install_dir:
+install:
 	mkdir -p $(INSTALL_DIR)
-
-$(INSTALL_DIR)/%: %
-	cp $< $@
-	chmod ugo+rx $@
-
-install: install_dir $(INSTALL_FILES)
+	cp $(WORKING_DIR)/{$(PYTHON_FILES)} $(INSTALL_DIR)
+	chmod ugo+rx $(WORKING_DIR)/{$(PYTHON_FILES)}
```

Comment:
By moving to `setup.py`, you should only need to run `$PYTHON setup.py install` to install the module in a place that's discoverable by your environment.

Reply:
I kept it this way to avoid making changes to the container.
@rdt12 Please confirm